nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-client-service/src/main/java/org/apache/nifi/distributed/cache/client/CacheClientChannelPoolFactory.java

```diff
@@ -70,7 +71,7 @@ public ChannelPool createChannelPool(final String hostname,
                                          final SSLContextService sslContextService,
                                          final VersionNegotiatorFactory factory) {
         final SSLContext sslContext = (sslContextService == null) ? null : sslContextService.createContext();
-        final EventLoopGroup group = new NioEventLoopGroup();
+        final EventLoopGroup group = new NioEventLoopGroup(new DefaultThreadFactory("DMCNioEventLoopGroup", true));
```

Comment:
This looks like a good opportunity to pass the Component Name and ID as the thread name prefix when instantiating the Pool Factory, along the lines of `DistributedMapCacheClient[id=UUID]`.

Reply:
Great suggestion, so I borrowed the pattern from [EventCacheServer](https://github.com/apache/nifi/blob/main/nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server/src/main/java/org/apache/nifi/distributed/cache/server/EventCacheServer.java#L98).  Thread names now look like "NettyDistributedMapCacheClient[43cf63eb-0188-1000-365c-d44c52e3677e]-1-3@23872"

Reply:
That's great, thanks @mosermw!